### ISSUE TITLE
export fails for recombined (with extraction/fusion) score

### ISSUE THREAD

**Issue by phdm** [Audiveris/audiveris#658]
Hello,

I have used extraction/fusion to replace the 5th page of a 6-page pdf score  by a copy of the 5th page hand-edited to erase the tiny border at the left created by the scan operation (see #652).  However the export operation fails with the following stack trace#657
```
INFO  []                      Book 2887 | Loading book C:\Users\phdm\Documents\Audiveris\Friends - I'll be there for you\I_ll_be_there_for_you-patched.omr
INFO  [I_ll_be_there_for_you-patched#5]                 SheetStub 1044 | Loaded /sheet#5/sheet#5.xml
INFO  [I_ll_be_there_for_you-patched]                 SheetStub 1044 | Loaded /sheet#1/sheet#1.xml
INFO  [I_ll_be_there_for_you-patched]                 SheetStub 1044 | Loaded /sheet#2/sheet#2.xml
INFO  [I_ll_be_there_for_you-patched]                 SheetStub 1044 | Loaded /sheet#3/sheet#3.xml
INFO  [I_ll_be_there_for_you-patched]                 SheetStub 1044 | Loaded /sheet#4/sheet#4.xml
INFO  [I_ll_be_there_for_you-patched]                 SheetStub 1044 | Loaded /sheet#6/sheet#6.xml
WARN  [I_ll_be_there_for_you-patched]               BookActions 2155 | Error in ExportBookTask java.lang.NullPointerException: Cannot invoke "java.util.List.iterator()" because the return value of "org.audiveris.omr.score.Score.getLogicalParts()" is null
java.lang.NullPointerException: Cannot invoke "java.util.List.iterator()" because the return value of "org.audiveris.omr.score.Score.getLogicalParts()" is null
        at org.audiveris.omr.sheet.rhythm.Voices.refineScore(Voices.java:339)
        at org.audiveris.omr.sheet.Book.reduceScores(Book.java:2082)
        at org.audiveris.omr.sheet.Book.transcribe(Book.java:2583)
        at org.audiveris.omr.sheet.Book.export(Book.java:634)
        at org.audiveris.omr.sheet.ui.BookActions$ExportBookTask.doInBackground(BookActions.java:2152)
        at org.audiveris.omr.sheet.ui.BookActions$ExportBookTask.doInBackground(BookActions.java:2125)
        at java.desktop/javax.swing.SwingWorker$1.call(SwingWorker.java:304)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
        at java.desktop/javax.swing.SwingWorker.run(SwingWorker.java:343)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
        at java.base/java.lang.Thread.run(Thread.java:833)
```
This happens both with development version f3b495c23e1f433201e56c1ec334dfc32c4fdd69 and 5.3.0 vanilla

**Comment by jazzbassoon** [comment]
I used the split and merge function to pull one movement out of a book, and that new book won't export because of the same error
`Error in ExportBookTask java.lang.NullPointerException: Cannot invoke "java.util.List.iterator()" because the return value of "org.audiveris.omr.score.Score.getLogicalParts()" is null`

**Comment by hbitteur** [comment]
I could reproduce this case: exporting a book right after a merge, leading to the same error message as yours.

The NullPointerException occurs because there is yet no logical parts defined in the score.

To populate the logical parts, all I had to do was to use a right-click, then in the popup menu select the menu item "`Score #1`", and then "`Collate parts`".
For the sake of conscience, I opened the logical parts editor just to check its content.
I could then export the book.

This manual action "fixes" the problem, but such workaround is not satisfactory.

Stay tuned...
